assets/js/content-script/content-script.js

```diff
@@ -66,6 +66,31 @@ function getContext(rules) {
   return values;
 }
 
+function inject(site) {
+  var injectionMethods = {
+    prepend: function(target, elem) {
+      target.parentNode.insertBefore(elem, target.firstChild);
+    },
+    append: function(target, elem) {
+      target.appendChild(elem);
+    },
+    replace: function(target, elem) {
+      target.innerHTML = elem;
+    }
+  };
+
+  if(!site.injection || !injectionMethods[site.injection.type]) {
+    console.log('injection abort');
```

Comment:
Actually this is some debug that souldn't be here, but this may be a useful thing to be know.